airbyte-ci/connectors/base_images/base_images/bases.py

```diff
@@ -19,6 +19,10 @@ class AirbyteConnectorBaseImage(ABC):
     Please do not declare any Dagger with_exec instruction in this class as in the abstract class context we have no guarantee about the underlying system used in the base image.
```

Comment:
Does this concern still apply? I assume that the commands we're using are so basic as to not be concerned, but the messaging is confusing